addons/website_google_map/static/src/js/website_google_map.js

```diff
@@ -88,7 +88,8 @@ function initialize_map() {
         if (odooPartnerData) { /* odoo_partner_data special variable should have been defined in google_map.xml */
             const markerPromises = [];
             for (var i = 0; i < odoo_partner_data.counter; i++) {
-                markerPromises.push(set_marker(odoo_partner_data.partners[i]));
+                markerPromises.push(set_marker(odoo_partner_data.partners[i]).catch((error)=>
```

Comment:
> Commit message: max 72 characters per line please

:point_up: 

> Missing spaces

You can probably write the line that way:
```javascript
const prom = set_marker(odoo_partner_data.partners[i])
    .catch(error => console.log(error));
markerPromises.push(prom);
```

Also, using `console.error` seems better? Or at least `warn`?